nbpackage/src/main/resources/org/apache/netbeans/nbpackage/rpm/rpm.spec.template

```diff
@@ -0,0 +1,23 @@
+Name:           ${RPM_PACKAGE}
+Version:        ${RPM_VERSION}
+Release:        0
```

Comment:
Should this default to 1?

Reply:
It depends on whom you ask :)
openSUSE: 0
Fedora: 1
Do you want me to change it to 1?

Reply:
Why can no-one agree?! :laughing: OK, keep as is (pretty much solely based on the fact that SUSE was my first distro 20+ years ago and therefore is probably correct!)

Reply:
Sorry for the late reply. I have been on holidays (and without a laptop).

Indeed :laughing:
OK, let's leave it as it is; 0.